jbpm-designer-backend/src/main/java/org/jbpm/designer/bpmn2/impl/Bpmn2JsonUnmarshaller.java

```diff
@@ -7278,7 +7279,12 @@ private String decodeAssociationValue(String s) {
         if (s == null || s.isEmpty()) {
             return s;
         }
-        return s.replaceAll("\\|\\|", "=").replaceAll("##", ",");
+        try {
+            return URLDecoder.decode(s, "UTF-8");
+        }
+        catch (UnsupportedEncodingException e) {
+            return s;
+        }
     }
```

Comment:
Is it always a URL that gets encoded here or could it be any String value that you want as UTF-8? Then I would do this instead: s.getBytes( Charset.forName( "UTF-8" ) ). This also has the advantage of not throwing a checked exception....so you can get rid of the catch block.